forms/includes/item-input.xbl

```diff
@@ -339,29 +339,104 @@
+
                 <h:div>
-                  <dcm:create nodeset="m:dimensions"
-                              label="Add dimensions"
-                              origin="xxf:instance('empty-instance')/m:meiHead/m:manifestationList/m:manifestation/m:itemList/m:item/m:physDesc/m:dimensions"/>
-                  <xf:repeat nodeset="m:dimensions" id="physdesc-dimensions">
-                    <h:div>
-                      <h:span class="fixed_width">Dimensions:</h:span>
-                      <xf:input ref=".">
-                        <xf:label>Value <h:a class="help">&#160;?<h:span class="comment">Numbers describing the physical dimensions 
-                        of the source, usually given as height x width, e.g. "23.5x34"</h:span></h:a></xf:label>
-                      </xf:input>
-                      <xi:include href="unit-select.xml" parse="xml"/>
-                      <dcm:attribute-editor ref="."/>                                                    
-                      <dcm:element-buttons  
-                          triggers="add remove" 
-                          nodeset="m:dimensions" 
-                          index="physdesc-dimensions"
-                          origin="xxf:instance('empty-instance')/m:meiHead/m:manifestationList/m:manifestation/m:itemList/m:item/m:physDesc/m:dimensions"/>
-                    </h:div>
-                  </xf:repeat>
+                    <dcm:create nodeset="m:dimensions" label="Add dimensions"
+                        origin="xxf:instance('empty-instance')/m:meiHead/m:manifestationList/m:manifestation/m:physDesc/m:dimensions"/>
+                    <xf:group ref="m:dimensions"> 
+                    
+                    <dcm:create nodeset="m:height" label="Add height"
+                        origin="xxf:instance('empty-instance')/m:meiHead/m:manifestationList/m:manifestation/m:physDesc/m:dimensions/m:height"/>
+                    <dcm:create nodeset="m:width" label="Add width"
+                        origin="xxf:instance('empty-instance')/m:meiHead/m:manifestationList/m:manifestation/m:physDesc/m:dimensions/m:width"/>
+                    <dcm:create nodeset="m:depth" label="Add depth"
+                        origin="xxf:instance('empty-instance')/m:meiHead/m:manifestationList/m:manifestation/m:physDesc/m:dimensions/m:depth"/>
+                    <dcm:create nodeset="m:dim" label="Add other dimension"
+                        origin="xxf:instance('empty-instance')/m:meiHead/m:manifestationList/m:manifestation/m:physDesc/m:dimensions/m:dim"/>
+
+                    <xf:repeat nodeset="m:height" id="physdesc-dimensions-height">
+                        <h:div>
+                            <h:span class="fixed_width">Dimensions:</h:span>
+                                <xf:input ref="@quantity">
+                                    <xf:label>Height <h:a class="help">&#160;?<h:span
+                                      class="comment">Numbers describing the physical
```

Comment:
I might be misreading this, but are you sure this is the right help comment for the Height label?